### PR TITLE
feat: Introducing hotkeys

### PR DESCRIPTION
**Pull request by patrickhempel**
I've implemented #37 

- s for start/stop sharing
- f for fullscreen current stream
- c copy room url
- h switch to next stream
- l switch to previous stream

### CODE REVIEW

File: ui/src/Room.tsx [hunk 1]
```diff
@@ -122,6 +129,42 @@ export const Room = ({
 
     const controlVisible = showControl || open || showMore || hoverControl;
 
+    useHotkeys('s', () => (hostStream ? stopShare() : share()), [hostStream]);
```
Comment (jmattheis) [inline ui/src/Room.tsx:132]: I don't think we need additional state like hostStream and clientStreams that is updated in a useEffect. state.hostStream should be used directly. Please do the same for clientStream.
```suggestion
    useHotkeys('s', () => (state.hostStream ? stopShare() : share()), [state.hostStream]);
```

**Review by jmattheis — CHANGES_REQUESTED** [review]
Thanks for your contribution. It works great!

I've added one little remark in a sub comment.

**Comment by patrickhempel** [comment]
Of course you are right. I've removed the unnecessary state usage.

File: ui/src/Room.tsx (hunk 1 above)
Comment (patrickhempel) [reply ui/src/Room.tsx]: You are right. I've removed the unnecessary stage usage.

**Review by jmattheis — APPROVED** [review]
Great, thanks!

**Pull request merged**